### ISSUE TITLE
Please tag a new version

### ISSUE THREAD

**Issue by uqs** [farrellm/scad-clj#49]
Hi there, I'm a clojure and leinigen noob working on a Dactyl keyboard and I need to shear a cylinder with multmatrix, but the latest release 0.5.3 pre-dates commit https://github.com/farrellm/scad-clj/commit/baa07380efa92316b5f27c5772fae9a6d487c68c

Can you please tag and release a new version so I can stick that into my project.clj?

Thanks a lot!

**Comment by farrellm** [comment]
Yes, I can tag a new release, though probably not until at least next week.

**Comment by dantheobserver** [comment]
also see that there's some other issues ironed out post 0.5.3.  in particular 07eca12b0706e01b2c59edcc384b93f2985b61cc, any word on that release?

**Comment by dantheobserver** [comment]
I was able to get through it by using 
https://github.com/LonoCloud/lein-voom/

and just creating an entry in project.clj as such then doing `lein voom build-deps` and `lein voom refresh`
```
                 ^{:voom {:repo "https://github.com/farrellm/scad-clj.git" :branch "master"}}
                 [scad-clj "0.5.4]
```
even though 0.5.4 isn't cut, this creates an entry for 0.5.4 based on the master branch, which has more fixes.
